tests/linear_algebra/test_inv.py

```diff
@@ -0,0 +1,152 @@
+
+import importlib
+import sys
+from unittest import TestCase
+
+import mock
+import numpy as np
+from numpy.testing import assert_allclose
+from pytest import mark
+
+from fastats.linear_algebra import inv
+
+
+class MatrixInverseValidator:
```

Comment:
I think that it looks nicer if you inherit `TestCase` here - that avoids some repetition down below. Alternatively, a pytest `fixture` could work here nicely. What do you think?

Reply:
If you inherit `TestCase` here the test will fail on this class - this is a mixin-only class, which requires `A` and `A_inv` to be set, so I agree with the current configuration.